app/models/site.rb

```diff
@@ -24,4 +28,11 @@ def instance
       first_or_create
     end
   end
+
+  private
+
+    def update_banner_config
+      # If a site banner_image uploaded, then update Sufia's banner_image to use it
+      Sufia.config.banner_image = banner_image? ? banner_image.url : nil
```

Comment:
Yeah, are we opening ourselves to weird cross-tenant contamination here, with multiple tenants relying on Sufia's in-memory configuration to set the background image?